packages/wrangler/src/proxy.ts

```diff
@@ -1,19 +1,14 @@
 import { connect } from "node:http2";
 import { createServer } from "node:http";
-import type {
-  Server,
-  IncomingHttpHeaders,
-  OutgoingHttpHeaders,
-  RequestListener,
-} from "node:http";
+import type { Server, IncomingHttpHeaders, RequestListener } from "node:http";
```

Comment:
NIT: I think we could actually go further and use `IncomingHttpHeaders` from `node:http2`?

Reply:
I'm actually not sure of the nuance here, especially because the server's made from the `http` module.